.github/workflows/main.yml

```diff
@@ -30,10 +30,6 @@ jobs:
             python-version: 3.9
             name: Python 3.9 with all non-visualization dependencies (except CASA)
             toxenv: py39-test-novis
-          - os: ubuntu-18.04
-            python-version: 3.6
```

Comment:
@keflavich  This is our only test case with CASA installed. Try bumping to 3.8 to keep it?